stratford/sass/_extra-child-theme.scss

```diff
@@ -168,6 +171,62 @@ a {
 				}
 			}
 
+			@include media(mobile-only) {
+				margin-top: .25em;
+
+				ul > li {
+					border: 0;
+					a {
+						border-color: $color_background_light;
+						border-style: solid;
+						border-width: 0 0 .125em;
+						width: auto;
+
+					}
+					ul li a	{
+						border-color: $color_background_light;
+						border-style: solid;
+						border-width: 0 0 .125em;
+						padding-left: 0.75em;
```

Comment:
I'm not sure if we need the side padding to indent the sub-menu items. When we have the `-`, we might not need it? Varia doesn't add the indentation.

<img width="453" alt="Screenshot 2019-10-24 at 12 00 04" src="https://user-images.githubusercontent.com/908665/67479586-48c4fb00-f656-11e9-9af1-35fe701cef2c.png">

<img width="453" alt="Screenshot 2019-10-24 at 12 04 17" src="https://user-images.githubusercontent.com/908665/67479664-70b45e80-f656-11e9-98f8-16434385ba4a.png">